workshops/forms.py

```diff
@@ -429,3 +436,71 @@ class HostForm(forms.ModelForm):
     class Meta:
         model = Host
         fields = ['domain', 'fullname', 'country', 'notes']
+
+
+class SwCEventRequestForm(forms.ModelForm):
+    captcha = ReCaptchaField()
+    workshop_type = forms.CharField(initial='swc', widget=forms.HiddenInput())
+
+    class Meta:
+        model = EventRequest
+        exclude = ('active', 'created_at', 'data_types', 'data_types_other',
+                   'attendee_data_analysis_level', 'fee_waiver_request', )
+        widgets = {
+            'approx_attendees': forms.RadioSelect(),
+            'attendee_domains': forms.CheckboxSelectMultiple(),
+            'attendee_academic_levels': forms.CheckboxSelectMultiple(),
+            'attendee_computing_levels': forms.CheckboxSelectMultiple(),
+            'travel_reimbursement': forms.RadioSelect(),
+            'admin_fee_payment': forms.RadioSelect(),
+        }
+
+
+class DCEventRequestForm(SwCEventRequestForm):
+    workshop_type = forms.CharField(initial='dc', widget=forms.HiddenInput())
+    understand_admin_fee = forms.BooleanField(
+        initial=False,
+        label='I understand the Data Carpentry\'s administration fee.',
+        help_text='There is a per-workshop fee for Data Carpentry to cover '
```

Comment:
We should probably have this for Software Carpentry workshops as well - please check wording with Maneesha.